README.md

```diff
@@ -35,7 +35,7 @@ providing the following features:
 
 The following features are high priority and planned for the very near future:
 
-* Ability to distribute builds to multiple build slaves
+* Ability to distribute builds to multiple build multiple build machines
```

Comment:
Now it says "multiple build multiple build"

Reply:
whoops, fixed!